### PR TITLE
ROX-15294: implement check-image-vulnerabilities workflow to use during upstream release

### PR DESCRIPTION
**Pull request by tommartensen**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results

## Testing Performed

Tested here https://github.com/stackrox/test-gh-actions/actions/runs/4252618039.

Output for 3.74.0-rc.4:

Screenshot 2023-02-23 at 14 24 59

### CODE REVIEW

**Comment by vikin91** [comment]
Looking at the example: https://github.com/stackrox/test-gh-actions/actions/runs/4253452545

I think that:
- There is no need for the Tag column in the output
- Instead I would use the second column for severity
- I would sort by severity first, and the image name second

**Comment by tommartensen** [comment]
@vikin91 

That would look like this:

https://github.com/stackrox/test-gh-actions/actions/runs/4282017796

Screenshot 2023-02-27 at 11 48 49

**Comment by vikin91** [comment]
> @vikin91
> 
> That would look like this:
> 
> https://github.com/stackrox/test-gh-actions/actions/runs/4282017796
> 
> Screenshot 2023-02-27 at 11 48 49

Removing the tag look good, but regarding the sorting, well, that is not what I was thinking about. I meant rather, sth like this:

```
central     high ...
collector   high ...
---
central     medium ...
collector   medium ...
---
scanner     low ...
central     low ...
```

Would that be doable?

**Comment by tommartensen** [comment]
@vikin91  like this? :) 

```
IMAGE                          SEVERITY   PACKAGE NAME         PACKAGE VERSION      VULNERABILITY                                                                   
=========================================================================================================================================================================================
central-db                     High       libksba              1.3.5-8.el8_6        RHSA-2023:0625: libksba security update (Important)                             
collector                      High       libksba              1.3.5-8.el8_6        RHSA-2023:0625: libksba security update (Important)                             
collector-slim                 High       libksba              1.3.5-8.el8_6        RHSA-2023:0625: libksba security update (Important)                             
main                           High       libksba              1.3.5-8.el8_6        RHSA-2023:0625: libksba security update (Important)                             
scanner                        High       libksba              1.3.5-8.el8_6        RHSA-2023:0625: libksba security update (Important)                             
scanner-db                     High       libksba              1.3.5-8.el8_6        RHSA-2023:0625: libksba security update (Important)                             
scanner-db-slim                High       libksba              1.3.5-8.el8_6        RHSA-2023:0625: libksba security update (Important)                             
scanner-slim                   High       libksba              1.3.5-8.el8_6        RHSA-2023:0625: libksba security update (Important)                             
central-db                     Medium     systemd              239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
central-db                     Medium     systemd-libs         239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
central-db                     Medium     systemd-pam          239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
collector                      Medium     systemd-libs         239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
collector-slim                 Medium     systemd-libs         239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
main                           Medium     systemd-libs         239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
scanner                        Medium     systemd-libs         239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
scanner-db                     Medium     systemd              239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
scanner-db                     Medium     systemd-libs         239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
scanner-db                     Medium     systemd-pam          239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
scanner-db-slim                Medium     systemd              239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
scanner-db-slim                Medium     systemd-libs         239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
scanner-db-slim                Medium     systemd-pam          239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
scanner-slim                   Medium     systemd-libs         239-68.el8_7.2       RHSA-2023:0837: systemd security and bug fix update (Moderate)                  
```

File: .github/workflows/scripts/check-image-vulnerabilities.py [hunk 1]
```diff
@@ -7,6 +7,14 @@
 
 QUAY_ORG="rhacs-eng"
 
+VULNERABILITY_LEVELS = {
```
Comment (vikin91) [inline .github/workflows/scripts/check-image-vulnerabilities.py:10]: Nit: It is actually a list of `VULNERABILITY_SEVERITIES`, but `SEVERITIES` would be also fine here.

**Review by vikin91 — APPROVED** [review]
Thanks for improving!

**Comment by tommartensen** [comment]
Here is a proof workflow run from master: https://github.com/stackrox/stackrox/actions/runs/4353547779.

**Pull request merged**